pkg/genlib/generator_interface.go

```diff
@@ -559,21 +531,22 @@ func bindLong(fieldCfg ConfigField, field Field, fieldMap map[string]any) error
 		return nil
 	}
 
+	min, _ := fieldCfg.Range.MinAsFloat64()
```

Comment:
we don't need to check for the value not being set here, since `min` will be returned as 0, and we'll take the maximum between `0` and `lowerBound>=0` and `max` will be `math.MaxFloat64` and we'll take the minimum between  `math.MaxFloat64` and `higherBound<=math.MaxFloat64`